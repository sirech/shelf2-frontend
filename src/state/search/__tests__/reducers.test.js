import reducer from '../reducers'

describe('search reducer', () => {
  it('returns the initial state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot()
  })
})
