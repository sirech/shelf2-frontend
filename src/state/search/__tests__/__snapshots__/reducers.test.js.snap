// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search reducer returns the initial state 1`] = `
Object {
  "entities": Object {
    "books": Object {},
  },
  "result": Array [],
}
`;
